src/Rating/Rating.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import ApiCalls from '../ApiCalls.js';
+import ReactStars from "react-rating-stars-component";
+import './Rating.css';
+
+class Rating extends Component {
```

Comment:
I think the refactor here to use the react-rating-stars package is a great idea & improves the UI.